addons/calendar/static/src/xml/base_calendar.xml

```diff
@@ -41,6 +41,12 @@
                 </div>
             </div>
         </t>
+        <t t-jquery=".o_cw_popover_fields_secondary" t-operation="attributes">
+            <attribute name="t-if">widget.isEventDetailsVisible()</attribute>
+        </t>
+        <t t-jquery=".o_cw_popover_edit" t-operation="attributes">
+            <attribute name="t-if">widget.isEventEditable()</attribute>
+        </t>
```

Comment:
I expect these `t-if` being defined in base template, so that you don't need `t-jquery`